client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -260,14 +260,13 @@ const RecurringPaymentsPlanAddEditModal = ( {
 						{ translate( 'Select renewal schedule' ) }
 					</FormLabel>
 					<FormSelect id="renewal_schedule" value={ editedSchedule } onChange={ onSelectSchedule }>
-						<option value="1 month">{ translate( 'Monthly' ) }</option>
-						<option value="1 year">{ translate( 'Yearly' ) }</option>
+						<option value="one-time">{ translate( 'One time sale (non-recurring)' ) }</option>
+						<option value="1 month">{ translate( 'Renew Monthly' ) }</option>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'renews monthly', { context: 'Informative text for renewals in /checkout'} )` *ES Score: 10*